plugins/MultiSites/angularjs/dashboard/dashboard.directive.less

```
@@ -277,6 +277,10 @@
     border-bottom: 5px solid @theme-color-brand;
   }
 
+  .multisites_icon {
+    display:inline-block;vertical-align:middle;
```

using multiple lines is easier to read:
```suggestion
    display: inline-block;
    vertical-align: middle;
```